docker/Dockerfile.j2

```
@@ -3,9 +3,15 @@
 
 {% set build_stage_base_image = "rust:1.45" %}
 {% if "alpine" in target_file %}
-{%   set build_stage_base_image = "clux/muslrust:nightly-2020-07-09" %}
-{%   set runtime_stage_base_image = "alpine:3.12" %}
-{%   set package_arch_name = "" %}
+{%   if "amd64" in target_file %}
+{%     set build_stage_base_image = "clux/muslrust:nightly-2020-07-09" %}
+{%     set runtime_stage_base_image = "alpine:3.12" %}
+{%     set package_arch_name = "" %}
+{%   elif "arm32v7" in target_file %}
+{%     set build_stage_base_image = "messense/rust-musl-cross:armv7-musleabihf" %}
+{%     set runtime_stage_base_image = "cmosh/alpine-arm" %}
```

```suggestion
{%     set runtime_stage_base_image = "cmosh/alpine-arm:3.6" %}
```

Let's add the version number like the others to try to make the images as reproducible as possible